lisa/features/nvme.py

```diff
@@ -78,7 +93,17 @@ def get_namespaces(self) -> List[str]:
         return namespaces
 
     def get_namespaces_from_cli(self) -> List[str]:
-        return self._node.tools[Nvmecli].get_namespaces()
+        namespaces_list = self._node.tools[Nvmecli].get_namespaces()
+        node_disk = self._node.features[Disk]
```

Comment:
The logic is repeated multiple times, it should be a new method like `_remove_nvme_os_disk`.

Reply:
Added a new method "_remove_nvme_os_disk()"